packages/react-components/react-dialog/src/components/DialogTitle/DialogTitle.cy.tsx

```diff
@@ -82,7 +82,7 @@ describe('DialogTitle', () => {
       mount(
         <Dialog modalType="alert">
           <DialogTrigger disableButtonEnhancement>
-            <Button>Open dialog</Button>
+            <Button id={dialogTriggerOpenId}>Open dialog</Button>
```

Comment:
why are all these ids necessary? I don't see them being used anywhere

Reply:
To make my life easier i just selected every single button at the same time and added them all these `ids`,  probably they're not necessary on most places..... but it was so many tests that this was the quickest way 🙈